src/sentry/features/manager.py

```diff
@@ -85,11 +96,21 @@ def has(self, name, *args, **kwargs):
         actor = kwargs.pop("actor", None)
         feature = self.get(name, *args, **kwargs)
 
+        if self._default_handler:
```

Comment:
I'm a little confused by this. Default normally implies the same thing as backup. "Use this value if no other value is set". This seems to be an override more than anything.